docs/products/postgresql/howto/migrate-aiven-db-migrate.rst

```diff
@@ -64,22 +70,37 @@ The output includes::
   User name for authentication with the server where to migrate data from
   ----
 
-These are the settings we'll use when configuring the migration in the next step.
-
-Perform the Migration
-'''''''''''''''''''''
-
-The following is the migration process:
-
-1. If you don't have a Aiven for PostgreSQL database already, run the following commands to create a couple of PostgreSQL services via :doc:`../../../tools/cli` substituting the parameters accordingly::
+  Variables
+  '''''''''
+  
+  You can use the following variables in the code samples provided:
+  
+  ==================      =======================================================================
+  Variable                Description
+  ==================      =======================================================================
+  ``SRC_HOSTNAME``        Hostname for source PostgreSQL connection
+  ``SRC_PORT``            Port for source PostgreSQL connection
+  ``SRC_DATABASE``        Database name for source PostgreSQL connection
+  ``SRC_USERNAME``        Username for source PostgreSQL connection
+  ``SRC_PASSWORD``        Password for source PostgreSQL connection
+  ``SRC_SSL``             SSL setting for source PostgreSQL connection
+  ``DEST_PG_NAME``        Name of the Aiven destination PostgreSQL service
+  ``DEST_PG_PLAN``        Aiven plan for the Aiven destination PostgreSQL service
+  ==================      =======================================================================
+  
+
+-> To perform the migration
```

Comment:
I'm not sure about the arrow in the title. Can you say a bit more about why you included this?

Reply:
I used this hihglight to indicate where the actual steps start. In sections above there are two other sets of instructions, which tell you how to put some prerequisites in place.

Reply:
I'd like to propose that as a convention for introducing actual steps, it helps readability and usually helps people's eyes jump directly to the steps.